lib/static_association.rb

```diff
@@ -40,6 +44,12 @@ def find_by_id(id)
       index[id]
     end
 
+    def find_by(**args)
+      args.any? or raise ArgumentError
```

Comment:
Does this automatically have a message like "wrong number of arguments (given 0, expected 1+)"?